services/datamanager/builtin/file_deletion.go

```diff
@@ -24,20 +24,24 @@ var (
 var errAtSizeThreshold = errors.New("capture directory has reached or exceeded disk usage threshold for deletion")
 
 func shouldDeleteBasedOnDiskUsage(ctx context.Context, captureDirPath string, logger logging.Logger) (bool, error) {
-	usage := diskusage.NewDiskUsage(captureDirPath)
+	statfs, err := diskusage.Statfs(captureDirPath)
+	if err != nil {
+		//nolint:nilerr
```

Comment:
What are your thoughts on adding a debug log for the error here if we hit it?

Reply:
good call, will add